extensions/parquet/table/src/brotliTest/java/io/deephaven/parquet/table/BroltiParquetTableReadWriteTest.java

```diff
@@ -0,0 +1,109 @@
+package io.deephaven.parquet.table;
+
+import io.deephaven.api.Selectable;
+import io.deephaven.base.FileUtils;
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.table.impl.TstUtils;
+import io.deephaven.engine.table.lang.QueryLibrary;
+import io.deephaven.engine.util.TableTools;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertTrue;
+
+public class BroltiParquetTableReadWriteTest {
```

Comment:
Shouldn't we be an `@Category(OutOfBandTest.class)`?

Reply:
No - the codec tests are basically instant, the read/write test should probably be split into two classes, and only run the slow tests as out of band?